### PR TITLE
Move emojis from food-marine to animal-marine (#753)

### PR DESCRIPTION
**Pull request by nedley**
See
- https://github.com/unicode-org/unicodetools/issues/753
- https://unicode-org.atlassian.net/browse/CLDR-14799

Also incorporate new CLDR localizations for Mx claus.

### CODE REVIEW

**Comment by markusicu** [comment]
Did you mean "clause" instead of "claus"?
Is there an action item for these changes? Or a CLDR ticket?

**Comment by markusicu** [comment]
> Did you mean "clause" instead of "claus"?

Ignore this question...
Changes lgtm; would be nice to document where the changes came from.

**Comment by nedley** [comment]
The name “Mx claus” is entirely up to CLDR. Even if the new capitalization is nonsensical (why “Mx claus” and not “Mx Claus”?) all I can do is file a ticket to find out what they were thinking.

**Comment by nedley** [comment]
https://unicode-org.atlassian.net/browse/CLDR-17701

**Comment by markusicu** [comment]
> Is there an action item for these changes? Or a CLDR ticket?

What I meant to ask here is where these changes are coming from.
Are they just the result of sucking in the latest CLDR data?
Is there an action item or CLDR ticket for the reclassification?

**Comment by markusicu** [comment]
And the changes mentioned in the title? “Move emojis from food-marine to animal-marine”
Are they also just a result of updating CLDR?

**Comment by nedley** [comment]
I made the change to emojiOrdering.txt as per #753, which is the headliner, the rest is generated.

**Comment by markusicu** [comment]
> I made the change to emojiOrdering.txt as per #753, which is the headliner,

Sorry, I totally missed that. I made it more explicit in the PR description now.

**Review by markusicu — APPROVED** [review]

**Comment by nedley** [comment]
Is there a more straightforward way to attach a PR (this one) to an existing issue (#753) to avoid this type of confusion in the future?

File: unicodetools/data/emoji/dev/emoji-test.txt [hunk 1]
```diff
@@ -1751,12 +1751,12 @@
 1F936 1F3FD                                            ; fully-qualified     # 🤶🏽 E3.0 Mrs. Claus: medium skin tone
 1F936 1F3FE                                            ; fully-qualified     # 🤶🏾 E3.0 Mrs. Claus: medium-dark skin tone
 1F936 1F3FF                                            ; fully-qualified     # 🤶🏿 E3.0 Mrs. Claus: dark skin tone
-1F9D1 200D 1F384                                       ; fully-qualified     # 🧑‍🎄 E13.0 mx claus
-1F9D1 1F3FB 200D 1F384                                 ; fully-qualified     # 🧑🏻‍🎄 E13.0 mx claus: light skin tone
-1F9D1 1F3FC 200D 1F384                                 ; fully-qualified     # 🧑🏼‍🎄 E13.0 mx claus: medium-light skin tone
-1F9D1 1F3FD 200D 1F384                                 ; fully-qualified     # 🧑🏽‍🎄 E13.0 mx claus: medium skin tone
-1F9D1 1F3FE 200D 1F384                                 ; fully-qualified     # 🧑🏾‍🎄 E13.0 mx claus: medium-dark skin tone
-1F9D1 1F3FF 200D 1F384                                 ; fully-qualified     # 🧑🏿‍🎄 E13.0 mx claus: dark skin tone
+1F9D1 200D 1F384                                       ; fully-qualified     # 🧑‍🎄 E13.0 Mx claus
```
Comment (macchiati) [inline unicodetools/data/emoji/dev/emoji-test.txt:1754]: FYI, it was an oversight with "claus", so that should be fixed in CLDR soon.

**Review by macchiati — APPROVED** [review]

**Comment by macchiati** [comment]
Once you merge and regenerate the emoji data for CLDR, let me know and I'll
make a PR on the CLDR side.

**Comment by nedley** [comment]
> Once you merge, let me know and I'll make a PR on the CLDR side.

Why is CLDR waiting on this? I already filed https://unicode-org.atlassian.net/browse/CLDR-17701.

**Comment by nedley** [comment]
Deleted my previous comment since I misunderstood Mark’s request.

**Comment by nedley** [comment]
@macchiati Merged.

**Comment by macchiati** [comment]
Great, I can take it from here.

**Comment by markusicu** [comment]
> Is there a more straightforward way to attach a PR (this one) to an existing issue (#753) to avoid this type of confusion in the future?

I didn't notice it in the title where it says “... to animal-marine (#753) #844” with the first number in black and the second in gray.

I suggest you mention the issue in the *description* instead.

If you want to formally connect them, you can say something like the following in the PR description:

Fixes #753

--> see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests

Also, if you want to link a PR to an issue, but not close that issue when merging the PR, you should be able to explicitly add the issue on the right side under "Development". (I haven't used that yet.)

**Comment by nedley** [comment]
Thanks. I’m much more used to the flow we use here, which doesn’t intermingle ticket and PR numbers.

**Pull request merged**